src/app/content/highlights/components/HighlightsPopUp.tsx

```diff
@@ -121,26 +122,31 @@ class HighlightsPopUp extends Component<Props> {
 
   public render() {
     return this.props.myHighlightsOpen ? (
-      <Styled.Modal>
-        <Styled.Mask>
-          <Styled.Wrapper
-            ref={this.popUp}
-            tabIndex='-1'
-            data-testid='highlights-popup-wrapper'
-          >
-            <Styled.Header>
-              <FormattedMessage id='i18n:toolbar:highlights:popup:heading'>
-                {(msg: Element | string) => msg}
-              </FormattedMessage>
-              <Styled.CloseIcon
-                data-testid='close-highlights-popup'
-                onClick={() => this.props.closeMyHighlights()}
-              />
-            </Styled.Header>
-            {this.props.user ? this.myHighlights() : this.loginForHighlights()}
-          </Styled.Wrapper>
-        </Styled.Mask>
-      </Styled.Modal>
+      <React.Fragment>
+        <ScrollLock overlay={false} mobileOnly={false} />
```

Comment:
looks like `ScrollLock` will render a mask, so you can probably use it inside the modal instead of the custom mask, just put it above the wrapper and i think it'll work. bonus, it has a click handler you can assign to close the modal

you also won't need the fragment